geowrangler/datasets/ookla.py

```diff
@@ -41,14 +68,208 @@ def list_ookla_files() -> dict:
     return keys
 
 
+# Cell
+
+
+def lookup_ookla_file(filename):
+    """Get OoklaFile for the given filename"""
+    ookla_files = L(list_ookla_files().items())  # use fastcore L as list
```

Comment:
This function can be expressed as 
```python
next((v for k,v in d.items() if k==filename), None)
```
which in my opinion is easier to read

Reply:
nice! will implement.